### ISSUE TITLE
Accessibility for sorting elements

### ISSUE THREAD

**Issue by eliyammine** [icflorescu/mantine-datatable#263]
**Describe the bug**
```
Expected the HTML found at $('.mantine-1a3sob1') to have no violations:

    <div class="mantine-Center-root mantine-1a3sob1" aria-label="Sorted ascending">

    Received:

    "Elements must only use allowed ARIA attributes (aria-allowed-attr)"

    Fix all of the following:
      aria-label attribute cannot be used on a div with no valid role attribute.

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.5/aria-allowed-attr?application=axeAPI
```
**To Reproduce**
Create a table with sortable:true on columns and run through jest-axe

**Expected behavior**
Expect no violations to be found
